zebra-rpc/src/methods/get_block_template_rpcs/types/subsidy.rs

```diff
@@ -13,9 +13,14 @@ use crate::methods::get_block_template_rpcs::types::zec::Zec;
 #[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub struct BlockSubsidy {
     /// An array of funding stream descriptions.
-    /// Always present, because Zebra returns an error for heights before the first halving.
-    #[serde(rename = "fundingstreams")]
-    pub funding_streams: Vec<FundingStream>,
+    /// Always present befoe NU6, Zebra returns an error for heights before the first halving.
+    #[serde(rename = "fundingstreams", skip_serializing_if = "Option::is_none")]
+    pub funding_streams: Option<Vec<FundingStream>>,
+
+    /// An array of lockbox stream descriptions.
+    /// Always present after NU6.
+    #[serde(rename = "lockboxstreams", skip_serializing_if = "Option::is_none")]
+    pub lockbox_streams: Option<Vec<FundingStream>>,
```

Comment:
Let's do the same check as the one in zcashd [here](https://github.com/zcash/zcash/pull/6912/files#diff-ccc24453c13307f815879738d3bf00eec351417537fbf10dde1468180cacd2f1R1008-R1013).

```suggestion
    /// Always present befoe NU6, Zebra returns an error for heights before the first halving.
    #[serde(rename = "fundingstreams", skip_serializing_if = "Vec::is_empty")]
    pub funding_streams: Vec<FundingStream>,

    /// An array of lockbox stream descriptions.
    /// Always present after NU6.
    #[serde(rename = "lockboxstreams", skip_serializing_if = "Vec::is_empty")]
    pub lockbox_streams: Vec<FundingStream>,
```

Reply:
Ok, thanks, i did this at https://github.com/ZcashFoundation/zebra/pull/8742/commits/e85d0f32c0c14a396d1faa408abbc613824eb73e